lte/gateway/c/core/oai/tasks/amf/amf_map.h

```diff
@@ -0,0 +1,181 @@
+
+#pragma once
+#include <pthread.h>
+
+#include <iostream>
+#include <unordered_map>
+#include <mutex>
+#include <string>
+
+#include "3gpp_23.003.h"
+
+namespace std {
```

Comment:
can we move this file to a common folder for reuse? how about ../core/oai/include/map.h and rename `amf_map` to `map`? that will also help us in the future for migrating other usages of hashtable to this

Reply:
How about moving both, amf_map.h and amf_map.cpp to a new folder "map" in ../core/oai/lib and renaming them to map.h and map.cpp respectively? If not, what would be an appropriate place for amf_map.cpp?

Reply:
Ignore the above comment. Will move the amf_map.h to ../core/oai/include/map.h

Reply:
This comment is addressed.